tests/Integration/goldens/redis/src/V1/Client/CloudRedisClient.php

```diff
@@ -131,7 +131,6 @@ private static function getClientDefaults()
             'apiEndpoint' => self::SERVICE_ADDRESS . ':' . self::DEFAULT_SERVICE_PORT,
             'clientConfig' => __DIR__ . '/../resources/cloud_redis_client_config.json',
             'descriptorsConfigPath' => __DIR__ . '/../resources/cloud_redis_descriptor_config.php',
-            'gcpApiConfigPath' => __DIR__ . '/../resources/cloud_redis_grpc_config.json',
```

Comment:
This shouldn't be removed - see this line, which should be changed to be `!== Transport::REST`:

https://github.com/googleapis/gapic-generator-php/blob/46dc31c550399de766d877038ec7d188f13fcfc9/src/Generation/GapicClientGenerator.php#L446

Reply:
I made this change and tested it locally and lots of golden removed this line. Shouldn't be "or" `Transport::REST`?

Reply:
Hmm that's weird, since the logic should be equivalent for all but the new case of gRPC only, which we just added. It should have gone from "when this is gRPC+rest" to "then this ISNT rest only"

Reply:
Ahhh found the issue. It was between the keyboard and the chair.

I forgot to flip the `===` to `!==`. My bad.

Reply:
I believe we still need to regenerate the Integration (golden) tests and push the result, as this change should be reverted.